examples/cordova-sample/MyApp/bin/sk_config_setup.rb

```diff
@@ -15,9 +15,20 @@
 storekit_file_path_for_xcproject = '../../RevenueCatTest.storekit'
 storekit_file_path_for_xcscheme = "../#{storekit_file_path_for_xcproject}"
 
-puts("Adding StoreKit Config to project")
+puts("Checking if we need to add the StoreKit Config to xcproject")
 project = Xcodeproj::Project.open(project_path)
-project.new_file(storekit_file_path_for_xcproject)
+already_exists = false
+for file in project.files
+    if file.path == storekit_file_path_for_xcproject
+        puts("StoreKit Config already exists in the project")
+        already_exists = true
+        break
+    end
+end
+if not already_exists 
+    puts("Adding StoreKit Config to project")
+    project.new_file(storekit_file_path_for_xcproject)
+end
```

Comment:
I think you can probably replace most of this block with `any?`, like

```ruby
already_exists = project.files.any? { |file| file.path == storekit_file_path_for_xcproject }

if already_exists
    puts("StoreKit Config file already exists")
    return
else
    puts("Adding StoreKit Config to project")
end
```

Reply:
I'm glad somebody here remembers ruby 🙃

Reply:
✅